recipe/conda_build_config.yaml

```diff
@@ -3,3 +3,5 @@
 # available on 10.13
 MACOSX_SDK_VERSION:        # [osx and x86_64]
   - "10.13"                # [osx and x86_64]
+channel_sources:
+  - conda-forge/label/mlir_tf,conda-forge/label/llvm_tf,conda-forge
```

Comment:
I already forgot what `mlir` stood for. Could you add a sentence or two as to what these are for.